packages/discovery-provider/plugins/pedalboard/apps/mri/scripts/seed-prod-db.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+sleep 10
```

Comment:
why the sleep?

gpt 🤷 

```
check_containers_health() {
    unhealthy_containers=$(docker ps --filter "health=unhealthy" --format "{{.Names}}")
    if [ -z "$unhealthy_containers" ]; then
        return 0 
    else
        return 1
    fi
}

while ! check_containers_health; do
    echo "Waiting for all containers to be healthy..."
    sleep 2
done

```

Reply:
ah that'll work, yeah basically docker thinks pg is healthy before it's "ready to accept connections"